packages/components/package.json

```diff
@@ -32,12 +31,14 @@
     "vite-plugin-solid": "^2.5.0"
   },
   "dependencies": {
-    "@tailwindcss/typography": "^0.5.9",
     "@photonhealth/sdk": "*",
+    "@tailwindcss/forms": "^0.5.3",
+    "@tailwindcss/typography": "^0.5.9",
     "clsx": "^1.2.1",
     "graphql": "^16.6.0",
     "graphql-tag": "^2.12.6",
     "jwt-decode": "^3.1.2",
+    "solid-heroicons": "^3.1.1",
```

Comment:
Should we lock these versions?

Reply:
yup, locked em up

Reply:
actually had to roll this back, ended up breaking elements, going to punt this with to https://github.com/Photon-Health/client/issues/178